client/app/bundles/course/forum/components/buttons/VotePostButton.tsx

```diff
@@ -47,15 +47,15 @@ const VotePostButton: FC<Props> = ({ post }) => {
     return (
       <div className="flex items-center">
         <IconButton
-          color="info"
+          color="success"
           onClick={(): void => handleVotePost(1)}
           title="Upvote"
         >
-          <ThumbUpOffAlt />
+          <ThumbUpOffAlt className="fill-green-600" />
```

Comment:
Why does ThumbUpOffAlt need to fill-green-600 but ThumbDownOffAlt not need any className?

Reply:
sorry it should not have been there.
I checked our codebase, and apparently for this kind of icon, most places use `IconButton` so I preserve it as it is, merely changing the color to become "success" for ThumbsUp and "error" for ThumbsDown